### PR TITLE
Completed css-positioning exercise.

### PR DESCRIPTION
**Pull request by sharon-tieu**
Screen Shot 2022-08-26 at 9 25 50 AM

Screen Shot 2022-08-26 at 9 37 36 AM

https://user-images.githubusercontent.com/103021766/186956682-6e9075c6-178e-4bb3-93a4-0356cb2bc49b.mov

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:7]: you want to have `position:relative` applied to this element

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  width: 300px;
+  position: absolute;
+  top: 170px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(21 20 20 / 75%));
+
+  /* your code here */
```
Comment (BrettAlbright) [inline css-positioning/absolute.css:42]: you want to have `position:absolute` applied to this element

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates

**Comment by sharon-tieu** [comment]
Screen Shot 2022-08-26 at 10 17 09 AM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**